lib/src/responder/responder.dart

```diff
@@ -10,7 +10,7 @@ class Responder extends ConnectionHandler {
   ISubscriptionResponderStorage storage;
   
   /// when true, the requester can't have higher permission than read
```

Comment:
Comment skew :-? ... suggested is to either remove in sync with read only flag removal or to adapt to new Permission derived constraints.